src/tabs/chat/ChatTabController.tsx

```diff
@@ -0,0 +1,69 @@
+import { TabController } from "../TabController";
+import { ChatTabState, Message } from "./ChatTabState";
+import { ChatAutoComplete } from "./ChatAutocomplete";
+
+export class ChatTabController extends TabController<ChatTabState> {
+    public toggleCommandView = () => {
+        var oldState = this.model.getState();
+        this.model.update({
+            commandMode: !oldState.commandMode
+        })
+    }
+
+    public execute = () => {
+        var command: string = this.model.getState().commandToSend;
+        if (this.model.getState().commandMode) {
+            if (this.IsChatCommand(command)) {
```

Comment:
Is there a particularly good reason why this command mode exists? Looks to me like you could remove this if statement completely and just leave what is in the else statement.

Reply:
I included it in case people preferred typing commands than using a dropdown to select the recipient. But if you don't think its needed, I'll remove it.